packager/react-packager/src/Server/index.js

```diff
@@ -701,6 +701,8 @@ class Server {
         telemetric: true,
       },
     );
+    const appRoot = process.cwd() + path.sep;
```

Comment:
I believe packager has the notion of project root, we should use that instead of relying on `cwd`. See `projectRoots` option above